io.github.neph_io.dsost.yml

```diff
@@ -0,0 +1,37 @@
+app-id: io.github.neph_io.dsost
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-23.08'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+
+command: /app/bin/launch.sh
+finish-args:
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
```

Comment:
either it supports wayland and it's `fallback-x11` or it is X11 only and then remove the wayland socket.